packages/marketplace/src/resources/locales/fr.json

```diff
@@ -42,6 +45,17 @@
       "reloadNow": "Recharger maintenant",
       "reloadLater": "Recharger plus tard"
     },
+    "backupModal": {
+      "title": "Importer/Exporter",
```

Comment:
The title in English is "Back up/Restore"

Reply:
Strange, i see this in the en file
![image](https://user-images.githubusercontent.com/20888017/199976382-3275d62c-6302-4842-a0a9-2fe42234ab5d.png)

Reply:
> Strange, i see this in the en file ![image](https://user-images.githubusercontent.com/20888017/199976382-3275d62c-6302-4842-a0a9-2fe42234ab5d.png)

That would be this, which is also `Back up/Restore`:
![image](https://user-images.githubusercontent.com/77577746/199978452-e6e4ad49-b4a5-4218-944e-771901f58ce3.png)

Reply:
You can reuse strings btw, in case you haven't noticed.

Reply:
Ok done.